setup.py

```diff
@@ -18,7 +18,7 @@
              'scripts/neomodel_remove_labels'],
     setup_requires=[
         'pytest-runner'] if any(x in ('pytest', 'test') for x in sys.argv) else [],
-    tests_require=['pytest>=6.2.4'],
+    tests_require=['pytest>=6.2.4', 'pytest>=6.2.4,<7.1.0;python_version<"3.7"'],
```

Comment:
A bit confused about this. Why do we need a version less than "3.7"?

Reply:
pytest>=7.1.0 dropped support for python 3.6. So if you run `python setup.py test` with python<=3.6, the constraint `"pytest<7.1.0` is necessary.